src/commcare_cloud/terraform/modules/postgresql/main.tf

```diff
@@ -1,5 +1,16 @@
 locals {
   create = "${(var.create && lookup(var.rds_instance, "create", true)) ? 1 : 0}"
+
+  version_parts = regex("^(?P<major>[0-9]+)(?:\\.(?P<minor>[0-9]+))?", var.rds_instance["engine_version"])
+  version_parts_number = {
+    major = tonumber(local.version_parts["major"])
+  }
+
+  uses_shorthand_major_version = local.version_parts_number["major"] >= 10
+
+  computed_major_engine_version = local.uses_shorthand_major_version ? local.version_parts["major"] : "${local.version_parts["major"]}.${local.version_parts["minor"]}"
+  major_engine_version = ""
```

Comment:
I'm dropping this text from the postgres docs https://www.postgresql.org/support/versioning/ just to provide context for others, as it helped me:

> Starting with PostgreSQL 10, a major version is indicated by increasing the first part of the version, e.g. 10 to 11. Before PostgreSQL 10, a major version was indicated by increasing either the first or second part of the version number, e.g. 9.5 to 9.6.
>
> Minor releases are numbered by increasing the last part of the version number. Beginning with PostgreSQL 10, this is the second part of the version number, e.g. 10.0 to 10.1; for older versions this is the third part of the version number, e.g. 9.5.3 to 9.5.4.

Reply:
I just learned that `regex` has been introduced in 0.12.7 version, this change is not working with terraform11. Making changes to support both versions. 
https://github.com/hashicorp/terraform/blob/v0.12.7/CHANGELOG.md